nixos/modules/services/misc/plex.nix

```diff
@@ -0,0 +1,90 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.plex;
+  plex = pkgs.plex;
+in
+{
+  options = {
+    services.plex = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = "Enable Plex Media Server";
+      };
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/plex";
+        description = "The directory where Plex stores its data files.";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "plex";
+        description = "User account under which Plex runs.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "plex";
+        description = "Group under which Plex runs.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    # Most of this is just copied from the RPM package's systemd service file.
+    systemd.services.plex = {
+      description = "Plex Media Server";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+      preStart = ''
+        mkdir -p "${cfg.dataDir}"
+        # If ${cfg.dataDir}/Resources doesn't exist, create it and populate it
+        # with the initial files.
+        test -d "${cfg.dataDir}/Resources" || {
+          echo "Initializing Resources directory."
+          cp -dr --no-preserve="ownership" \
+              "${plex}/usr/lib/plexmediaserver/base-resources" \
+              "${cfg.dataDir}/Resources"
+          chmod -R u+w "${cfg.dataDir}/Resources"
+        }
+        chown -R ${cfg.user}:${cfg.group} "${cfg.dataDir}"
+     '';
+      serviceConfig = {
+        Type = "simple";
+        User = "plex";
+        Group = "plex";
```

Comment:
It really doesn't, but it takes little extra effort to make it configurable, so I don't see any reason not to.